polyfill.test262.mjs

```diff
@@ -0,0 +1,55 @@
+import path from "path";
+import tape from "tape";
+import Test262Stream from "test262-stream";
+import NodeAgent from "eshost/lib/agents/node.js";
+import { readFileSync } from "fs";
+
+const POLYFILL = readFileSync(
+	new URL("./polyfill.js", import.meta.url),
+	"utf-8"
+);
+
+const agent = new NodeAgent({
+	hostPath: process.argv0,
+	transform(source) {
+		return POLYFILL + ";\n" + source;
+	},
+});
+
+const testsPaths = [
+	["Array", "toReversed"],
+	["Array", "toSorted"],
+	["Array", "toSpliced"],
+	["Array", "with"],
+	["Array", "Symbol.unscopables/change-array-by-copy.js"],
+	["TypedArray", "toReversed"],
+	["TypedArray", "toSorted"],
+	["TypedArray", "toSpliced"],
```

Comment:
I’m away from my laptop so harder to check out changes. So I might be missing something obvious. How comes there isn’t a TypedArray + With pair in this list?

Reply:
Ah just seen the PR description. They aren’t written yet

Reply:
Yeah I still have to write them (It's not hard, it's just boring :joy:).